### ISSUE TITLE
Unable to Run newly created project

### ISSUE THREAD

**Issue by gfo2007** [volkanceylan/Serenity#3323]
Hi,
After creating a new project, I updated the project to the latest Serenity .Web and Serenity.CodeGenerator with 

Update-Package Serenity.Web
Update-Package Serenity.CodeGenerator

Everything updated to version 3.5.5

However, when I tried to run the project, the error below showed up.

![error serenity](https://user-images.githubusercontent.com/14317420/37002943-3b07d15a-20c3-11e8-8835-8692cbec682e.PNG)

**Comment by ga5tan** [comment]
did u check loaderException? For me it was mono.cecil.dll in tools having bad version
I copied it over from packages\Serenity.Web\net45 ( or something like that) and that fixed the problem

**Comment by gfo2007** [comment]
It's the same thing. Where did you copy the right version?

**Comment by brunobola** [comment]
Please update the serenity.web.tooling and try.

**Comment by gfo2007** [comment]
Hi,
After updating serenity.web.tooling, another error showed up when I tried to transform t4 templates. Below is the error.

Severity	Code	Description	Project	File	Line	Suppression State
Error		Compiling transformation: 'InterfaceImplementation' does not contain a definition for 'FullName' and no extension method 'FullName' accepting a first argument of type 'InterfaceImplementation' could be found (are you missing a using directive or an assembly reference?)		J:\Visual Studio Projects\hails\hails\hails.Web\Imports\ClientTypes\..\CodeGenerationHelpers.ttinclude	167

**Comment by brunobola** [comment]
Please do a upgrade --reinstall and see if it helps... don't forget to close the VS ope nthe solution again and then build/T4/build and test

**Comment by ga5tan** [comment]
This may very well be the "[Breaking Change] need to replace "externalType.Interfaces.Add(intf.FullName)" with "externalType.Interfaces.Add(intf.InterfaceType.FullName)" in CodeGenerationHelpers.ttinclude

do as it says.
I did it before trying to run project, so dunno how it manifests, but sounds like it.

**Comment by gfo2007** [comment]
Ok thanks to you @ga5tan and @brunobola. I combined your suggestions to fix my issue. 

1. After updating my project to the newest serenity.web and serenity.codegenerator packages, I had to update serenity.web.tooling as well.

2. I opened CodeGenerationHelpers.ttinclude and updated the line of code indicated by @ga5tan in the comment. I rebuilt solution, transformed t4 Templates and run and viola, all errors were gone.
